lib/commands/command-help.bash

```diff
@@ -14,20 +14,17 @@ EOF
 }
 
 asdf_extension_cmds() {
-  local plugins_path ext_cmd_path ext_cmds plugin
+  local plugins_path plugin_path ext_cmd_path ext_cmds plugin
   plugins_path="$(get_plugin_path)"
-  # use find instead of ls -1
-  # shellcheck disable=SC2012
-  for plugin in $(ls -1 "$plugins_path" 2>/dev/null | sed "s#^$plugins_path/##"); do
```

Comment:
I was able to remove both `ls` and `sed` from this line and replace the `sed` command with a simpler `basename` command. I'm considering adding `ls` to the banned commands list as I see it used in some anti-patterns in our codebase. To me it seems like listing files is done better with commands like `find`, `grep`, and the occasional globbing.